src/lobster/callbacks/_umap_visualization_callback.py

```diff
@@ -498,6 +498,55 @@ def _run_umap(self, embeddings: np.ndarray) -> np.ndarray:
 
         return reducer.fit_transform(embeddings)
 
+    def _save_embeddings(
+        self,
+        embeddings: dict[str, torch.Tensor] | torch.Tensor,
+        output_file: str | UPath,
+        model_name: str = "model",
```

Comment:
could something more descriptive be pulled from a variable to be used here as a default if the input is `None` ?